WordPress/Classes/ViewRelated/Post/PostCardCell.swift

```diff
@@ -278,6 +286,10 @@ class PostCardCell: UITableViewCell, ConfigurablePostView {
         retryButton.setImage(Gridicon.iconOfType(.refresh, withSize: CGSize(width: 18, height: 18)), for: .normal)
         retryButton.isHidden = true
 
+        cancelAutoUploadButton.setTitle(NSLocalizedString("Cancel", comment: "Labal for the auto-upload cancelation button in the post list. Tapping will prevent the app from auto-uploading the post."),
```

Comment:
misspelling: Labal should be Label

Reply:
Good eye. Fixed in ae0da70ca240835c98e832d5b02d184e3b4964a7.